DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -54,10 +54,9 @@ FROM Applications
                   INNER JOIN Sections ON Sections.ApplicationID = Applications.ApplicationID

```

Comment:
I assume you've tested the specific AC which was failing earlier, i.e. enrolling on a course which you've removed? 🙂

Reply:
Yes, seems to be working for that case now.

Reply:
Great 🙂